janitor/io.py

```diff
@@ -462,6 +500,21 @@ def xlsx_cells(
         path = load_workbook(
             filename=path, read_only=read_only, keep_links=False
         )
+    if engine not in {"pandas", "polars"}:
+        raise ValueError("engine should be one of pandas or polars.")
+    base_engine = pd
```

Comment:
@samukweku what are your thoughts on assigning `base_engine` whenever we dispatch differently between polars and pandas? Does it make sense for us to use one pattern instead?